### PR TITLE
Smtp configuration

### PR DESCRIPTION
**Pull request by pritt20**
New SMTP configuration document

### CODE REVIEW

**Comment by paolaperaza** [comment]
Self-note to make sure this doc covers issues customer had in this support ticket: https://astronomer.zendesk.com/agent/tickets/2897

File: enterprise/v0.16/06_manage-astronomer/09_smtp-configuration.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	reply: “signedemail@mydomain.com”
+	smtpUrl: “smtps://apikey:SG.sometoken@smtp.sendgrid.net:465/?pool=true”
+```
+​
+#### Mailgun:
+​
+```
+email:
+        enabled: true
+        smtpUrl:smtps://postmaster%40mg.astronomer.io:daf255573ca4f22c699c6c0c4ab2dfb0@smtp.mailgun.org/?pool=true
```
Comment (ianstanton) [inline enterprise/v0.16/06_manage-astronomer/09_smtp-configuration.md:57]: I would mock this up as an example vs using our mailgun account info

**Review by ianstanton — CHANGES_REQUESTED** [review]
@pritt20 A lot of good examples here. Thank you for throwing this together. We'll be able to add more examples as we run into them! My only request is to mock up the mailgun example with dummy account info.

File: enterprise/v0.16/06_manage-astronomer/09_smtp-configuration.md (hunk 1 above)
Comment (pritt20) [reply enterprise/v0.16/06_manage-astronomer/09_smtp-configuration.md]: Thanks for pointing that out @ianstanton !! I've made the required changes under mailgun smtp example url

File: enterprise/v0.16/06_manage-astronomer/09_smtp-configuration.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        smtpUrl:smtps://xyz%40example.com:password@smtp.mailgun.org/?pool=true
+```
+​
+#### Office365:
+​
+```
+email:
+        reply: "xyz@example.com"
+        enabled: true
+        smtpUrl: smtp://xyz%40example.com:password@smtp.office365.com:587/?requireTLS=true
```
Comment (ianstanton) [inline enterprise/v0.16/06_manage-astronomer/09_smtp-configuration.md:66]:
```suggestion
email:
  reply: "xyz@example.com"
  enabled: true
  smtpUrl: smtp://xyz%40example.com:password@smtp.office365.com:587/?requireTLS=true
```

File: enterprise/v0.16/06_manage-astronomer/09_smtp-configuration.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+ ```       
+        
+#### Custom SMTP-relay:
+​
+```
+email:
+	enabled: true
+	reply: "xyz@example.com"
+	smtpUrl: smtp://smtp-relay.example.com:25/?ignoreTLS=true
```
Comment (ianstanton) [inline enterprise/v0.16/06_manage-astronomer/09_smtp-configuration.md:75]:
```suggestion
email:
  enabled: true
  reply: "xyz@example.com"
  smtpUrl: smtp://smtp-relay.example.com:25/?ignoreTLS=true
```

**Review by ianstanton — CHANGES_REQUESTED** [review]
Some indentation suggestions

**Review by ianstanton — APPROVED** [review]
@pritt20 content looks good to me. Assigning @jwitz as reviewer for formatting / editing purposes!

**Comment by ianstanton** [comment]
@jwitz @paolaperaza Do you guys think this should be merged and posted to our website? I feel it is useful.

**Comment by jwitz** [comment]
@ianstanton apologies for falling behind on this PR! definitely agree this is good info to have in docs, I just need some extra time to edit this and strategize over how to best include this info in our Enterprise setup. 

For instance: right now we have SMTP as a [prerequisite](https://www.astronomer.io/docs/enterprise/v0.23/install/aws/install-aws-standard#prerequisites), should this be its own step within the installation guide vs. an entirely separate guide? And how do we tell people to add these variable YAML objects to our example Helm chart? Gonna muse over this and post my plan here once it's finalized.

**Comment by jwitz** [comment]
Hi @pritt20 , thank you for writing this. I'm going to close this PR, as I've edited your content and found a place for it in our Enterprise installation guides over at PR https://github.com/astronomer/docs/pull/247. I'd appreciate your review over there!

**Pull request closed without merging**